validateFileNames.js

```diff
@@ -0,0 +1,94 @@
+const shelljs = require('shelljs');
+
+// Regex that defines our file and directory naming convention
+const camelCasedFileNameRegex = /^[.]?([a-z])+([0-9]|[a-zA-Z]|[.])*$/;
+
+// Files and directories that are exempt from the naming convention
+const ignoredFilesAndDirectories = [
+  'README.md',
+  'Dockerfile',
+  'LICENSE.md',
+  'customTypings',
+  'typings',
+];
```

Comment:
I'm thinking of reading this array of exempt files from a config file that exists in `hollowverse/hollowverse` itself or where ever this script is called from. That will make adding more files convenient, but it will add complexity to the setup of this script. What do you think?